Sources/WebSockets/Connection/WebSockets+Hash.swift

```diff
@@ -29,9 +29,10 @@ extension WebSocket {
          the |Sec-WebSocket-Accept| header field.
     */
     public static func exchange(requestKey: String) -> String {
-        let combination = requestKey.bytes.trimmed([.space]).string + hashKey
-        let shaBytes = SHA1.calculate(combination)
-        let hashed = shaBytes.base64
+        var combination = requestKey.bytes.trimmed([.space]).array + hashKey.bytes
+        var digest = Bytes(repeating: 0, count: Int(SHA_DIGEST_LENGTH))
```

Comment:
For instance there is also `SHA1_DIGEST_LENGTH` which I guess is the same? This stuff is trickier.